ci/test/00_setup_env_win64.sh

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env bash
 #
-# Copyright (c) 2019-2022 The Bitcoin Core developers
+# Copyright (c) 2019-present The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 export LC_ALL=C.UTF-8
 
 export CONTAINER_NAME=ci_win64
-export CI_IMAGE_NAME_TAG=ubuntu:22.04  # Check that Jammy can cross-compile to win64
+export CI_IMAGE_NAME_TAG="docker.io/amd64/ubuntu:22.04"  # Check that Jammy can cross-compile to win64
```

Comment:
Same change in `cirrus.sh`?

Reply:
Does the following log:
```
#3 [internal] load metadata for docker.io/library/ubuntu:jammy
```
look OK?

Reply:
I was expecting to see:
```
#3 [internal] load metadata for docker.io/amd64/ubuntu:22.04
```

The PR description claims that this is a bugfix, but changes do not affect CI builds at all, according to the logs.

I'm confused a bit about that.

Reply:
> Currently the task will fail if run on non-`x86_64`.

I expect it still fail with this PR with a non-`x86_64` Cirrus worker.